voluncheer/.ebextensions/django.config

```diff
@@ -5,6 +5,9 @@ container_commands:
   02_pull_data:
     command: "source /var/app/venv/*/bin/activate && python3 manage.py pull_nyc_charities"
     leader_only: true
+  04_load_badges:
+    command: "source /var/app/venv/*/bin/activate && python3 manage.py load_badge_data"
```

Comment:
Just FYI, Joe has a third command, so setting mine to 04 preemptively.

Reply:
Wouldn't there be redundent data in RDS if you trigger this command everytime you deploy? Would it be better if it's run once manually on aws elastic beanstalk console?

Reply:
It only creates the badges if they don't already exist, so most of the time nothing will happen. But if we need to rebuild the develop or main dbs, then this will take care of it automatically